cmd/jujud/updateseries/updateseries.go

```diff
@@ -118,38 +127,76 @@ func (c *UpdateSeriesCommand) Run(ctx *cmd.Context) error {
 		return err
 	}
 
+	jujuVersion, err := agent.GetJujuVersion(c.machineAgent, c.dataDir)
+	if err != nil {
+		return err
+	}
+
+	var (
+		startedUnitNames   []string
```

Comment:
You don't need these declarations - the first assignment to them (line 152) is doing a := which will create the variables, and then you only use them inside that if statement.